openedx/features/course_experience/views/latest_update.py

```diff
@@ -0,0 +1,47 @@
+"""
+View logic for handling latest course updates.
```

Comment:
I like the new name `update_message_fragment`, but I feel like somewhere we need to explain the difference between this and the welcome message fragment, because they seem like they serve the same purpose.

Could you add the following here?:
```

Although the welcome message fragment also displays the latest update,
this fragment dismisses the message for a limited time so new updates
will continue to appear, where the welcome message gets permanently
dismissed.
```